modules/mono/doc_classes/GodotSharp.xml

```diff
@@ -11,26 +14,30 @@
 			<return type="void">
 			</return>
 			<description>
-				Attaches the current thread to the mono runtime.
+				Attaches the current thread to the Mono runtime.
 			</description>
 		</method>
 		<method name="detach_thread">
 			<return type="void">
 			</return>
 			<description>
-				Detaches the current thread from the mono runtime.
+				Detaches the current thread from the Mono runtime.
 			</description>
 		</method>
 		<method name="get_domain_id">
 			<return type="int">
 			</return>
 			<description>
+				Returns the engine MonoDomain's ID.
+				[b]Note:[/b] The Mono runtime must be initialized for this method to work (use [method is_runtime_initialized] to check). If the Mono runtime isn't initialized at the time this method is called, the engine will crash.
```

Comment:
It's weird that calling a getter would crash the engine.
@neikeq I'd suggest changing this to a simple `ERR_FAIL_COND`. (Same for other occurrences.)

The engine should *not* crash when using the public scripting API.

Reply:
Since this method returns an integer, should it return `-1` in case an error occurs?

Reply:
I guess, as long as it's not a valid MonoDomain ID.